### DISCUSSION TITLE
1-0 of 1 notifications

### DISCUSSION THREAD

**Discussion by Gnimuc** [community/community#58545] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

![Screenshot 2023-06-20 at 20 04 55](https://github.com/community/community/assets/7437173/0f2f47c3-a258-4fbb-8e24-b0a60949e826)

Anyone know how to clear this notification?

**Comment by airtower-luna** [comment, marked as answer]
I've used this script when I had the same bug a while ago: https://gist.github.com/airtower-luna/4f371c7d8799632ba5b9ed855039a484

Mind that it can only mark the notification as read (so the "new notification" indicator should be gone), not actually remove the notification. :sweat_smile:

**Reply by Gnimuc** [reply]
Thanks!
